slack-api-model/src/main/java/com/slack/api/model/block/CallBlock.java

```diff
@@ -0,0 +1,50 @@
+package com.slack.api.model.block;
+
+import com.slack.api.model.CallParticipant;
+import com.slack.api.model.admin.AppIcons;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * https://api.slack.com/reference/messaging/blocks#call
+ */
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CallBlock implements LayoutBlock {
+    public static final String TYPE = "call";
+    private final String type = TYPE;
+    private String blockId;
+    private String callId;
+    private Boolean apiDecorationAvailable;
+    private Map<String, Call> call;
+
+    @Data
+    public class Call {
```

Comment:
A `call` object in a `call` block is far different from the one in `calls.*` API method response. That's why I've decided to have another class here, besides `com.slack.api.model.Call`.

### `calls.info` response

```json
"call": {
  "id": "R111",
  "date_start": 1590201051,
  "external_unique_id": "6082c846-5c1c-4505-a01a-c120b3da2f97",
  "join_url": "https://www.example.com/updated-call",
  "date_end": 1590201285,
  "channels": [
    "C111"
  ],
  "external_display_id": "ext-display-id-111",
  "title": "Test call",
  "desktop_app_join_url": "https://www.example.com/updated-call-2",
  "users": [
    {
      "slack_id": "USLACKBOT"
    },
    {
      "external_id": "anon-111",
      "avatar_url": "https://assets.brandfolder.com/pmix53-32t4so-a6439g/original/slackbot.png",
      "display_name": "anonymous user 1"
    },
    {
      "external_id": "anon-222",
      "avatar_url": "https://assets.brandfolder.com/pmix53-32t4so-a6439g/original/slackbot.png",
      "display_name": "anonymous user 2"
    },
    {
      "slack_id": "U111"
    }
  ]
}
```

### `call` block in a message

```json
"call": {
  "v1": {
    "id": "R111",
    "app_id": "A111",
    "app_icon_urls": {
      "image_32": "https://avatars.slack-edge.com/2019-02-23/559409256389_407beb3e6e85221800d7_32.jpg",
      "image_36": "https://avatars.slack-edge.com/2019-02-23/559409256389_407beb3e6e85221800d7_36.jpg",
      "image_48": "https://avatars.slack-edge.com/2019-02-23/559409256389_407beb3e6e85221800d7_48.jpg",
      "image_64": "https://avatars.slack-edge.com/2019-02-23/559409256389_407beb3e6e85221800d7_64.jpg",
      "image_72": "https://avatars.slack-edge.com/2019-02-23/559409256389_407beb3e6e85221800d7_72.jpg",
      "image_96": "https://avatars.slack-edge.com/2019-02-23/559409256389_407beb3e6e85221800d7_96.jpg",
      "image_128": "https://avatars.slack-edge.com/2019-02-23/559409256389_407beb3e6e85221800d7_128.jpg",
      "image_192": "https://avatars.slack-edge.com/2019-02-23/559409256389_407beb3e6e85221800d7_192.jpg",
      "image_512": "https://avatars.slack-edge.com/2019-02-23/559409256389_407beb3e6e85221800d7_512.jpg",
      "image_1024": "https://avatars.slack-edge.com/2019-02-23/559409256389_407beb3e6e85221800d7_1024.jpg",
      "image_original": "https://avatars.slack-edge.com/2019-02-23/559409256389_407beb3e6e85221800d7_original.jpg"
    },
    "date_start": 1590201051,
    "active_participants": [],
    "all_participants": [
      {
        "slack_id": "USLACKBOT"
      },
      {
        "external_id": "anon-111",
        "avatar_url": "https://assets.brandfolder.com/pmix53-32t4so-a6439g/original/slackbot.png",
        "display_name": "anonymous user 1"
      },
      {
        "external_id": "anon-222",
        "avatar_url": "https://assets.brandfolder.com/pmix53-32t4so-a6439g/original/slackbot.png",
        "display_name": "anonymous user 2"
      },
      {
        "slack_id": "U111"
      }
    ],
    "display_id": "ext-display-id-111",
    "join_url": "https://www.example.com/updated-call",
    "desktop_app_join_url": "https://www.example.com/updated-call-2",
    "name": "Test call",
    "created_by": "U8P5K48E6",
    "date_end": 1590201285,
    "channels": [
      "C111"
    ],
    "is_dm_call": false,
    "was_rejected": false,
    "was_missed": false,
    "was_accepted": false,
    "has_ended": true
  }
}
```

Reply:
Quite a different payload